vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_sku.cc

```diff
@@ -154,21 +155,178 @@ void ContributionSKU::Start(
     return;
   }
 
-  auto get_callback = std::bind(&ContributionSKU::GetContributionInfo,
+  auto get_callback = std::bind(&ContributionSKU::PrepareTokens,
       this,
       _1,
+      _2,
       item,
       *wallet,
       callback);
 
-  ledger_->GetContributionInfo(contribution_id, get_callback);
+  GetContributionInfoAndUnblindedTokens(contribution_id, get_callback);
```

Comment:
this will not work as only when you are in `ContributionSKU::Completed` you actually get tokens for SKU contribution. Actual contribution is started in `ContributionSKU::CredsStepSaved`. For Brave SKU we always get tokens on the fly as part of contribution flow.

Reply:
OK, will just table SKU changes for now.